PowerUp/app/src/main/AndroidManifest.xml

```diff
@@ -18,35 +19,44 @@
         </activity>
         <activity
             android:name=".GameActivity"
-            android:label="@string/app_name" />
+            android:label="@string/app_name"
+            android:theme="@style/AppBaseTheme"/>
         <activity
             android:name=".AvatarRoomActivity"
-            android:label="@string/app_name" />
+            android:label="@string/app_name"
+            android:theme="@style/AppBaseTheme"/>
         <activity
             android:name=".AvatarActivity"
-            android:label="@string/app_name" />
+            android:label="@string/app_name"
+            android:theme="@style/AppBaseTheme"/>
         <activity
             android:name=".MapActivity"
             android:label="@string/app_name"
-            android:launchMode="singleTop" />
+            android:launchMode="singleTop"
+            android:theme="@style/AppBaseTheme"/>
         <activity
             android:name=".CompletedSceneActivity"
-            android:label="@string/app_name" />
+            android:label="@string/app_name"
+            android:theme="@style/AppBaseTheme"/>
         <activity
             android:name=".GameOverActivity"
-            android:label="@string/app_name" />
+            android:label="@string/app_name"
+            android:theme="@style/AppBaseTheme"/>
 
         <uses-library android:name="android.test.runner" />
 
         <activity
             android:name=".DressingRoomActivity"
-            android:label="@string/app_name"/>
+            android:label="@string/app_name"
+            android:theme="@style/AppBaseTheme"/>
         <activity
             android:name=".SelectFeaturesActivity"
-            android:label="@string/app_name"/>
+            android:label="@string/app_name"
+            android:theme="@style/AppBaseTheme"/>
         <activity
             android:name=".ScenarioOverActivity"
-            android:label="@string/app_name"/>
+            android:label="@string/app_name"
+            android:theme="@style/AppBaseTheme"/>
```

Comment:
There is no need to change the theme of each activity and can be done by changing the base theme of the application in the manifest. As the current AppTheme inherits from the AppBaseTheme where you have specified the changes, I do not think specification is required in each activity declaration.